package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,22 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Similar to the check for atomtypes function
+        if not any(elements):
+            # Can further add a check_cg_atom(elements) function in guessers.py to check for CG atom.
+            elements = guess_atom_element(elements)
+            if elements == '':
+                warnings.warn("Element record found to be non-physical.")
+                #Nomenclature that X will denote any non-physical element.
+                elements = 'X'
```

Comment:
From #2553 I think the consensus is that `element` should be a False-y value if it's non-physical, not a special name.

Reply:
Sounds good. I thought of it as 3 conditionals. 1. In case it is an existing and valid element. 2. The element exists, but is not valid. .i.e. non-physical maybe because of the limitations of the tables or being a CG type? 3. It does not exist. 
Should I just have 2 instead?